cqrs-user-command-service/src/test/java/dev/amir/usercommand/application/usecase/UserUseCasesTest.java

```diff
@@ -127,7 +127,7 @@ void test_ChangePassword() {
     @Test
     void test_DeleteUser() {
         User userResponse = new User();
-        userResponse.setId(new UserId());
+        userResponse.setId(new UserId()); //TODO preguntar si esto sirve?
```

Comment:
i think that this lines are not used, Should they be removed?

Reply:
delete `userResponse`